src/app/onion/learning-object-builder/learning-object-builder.component.ts

```diff
@@ -124,11 +124,15 @@ export class LearningObjectBuilderComponent implements OnInit, OnDestroy {
       // if name parameter found, instruct store to fetch full learning object
       if (id) {
         this.store.fetch(id).then(obj => {
-          // redirect user to dashboard if the object is in the working stage
-          if (!obj.status.includes('waiting') && !obj.status.includes('review') && !obj.status.includes('proofing')) {
-            this.setBuilderMode(obj);
+          if (obj.status === LearningObject.Status.RELEASED) {
```

Comment:
Why not just add this check to the if statement below instead of an if-else here?

Reply:
If it is added to the if statement below, then the user will still be allowed to access editing via the builder if they are an editor/admin, I believe.

Reply:
That's actually what we want, admins and editors do have privileged access to open other user's objects in the builder.